device/iot-device-samples/module-method-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/ModuleMethodSample.java

```diff
@@ -23,12 +23,8 @@
             + "2. (mqtt | amqps | amqps_ws | mqtt_ws)\n";
 
     private static String SAMPLE_USAGE_WITH_WRONG_ARGS = "Expected 2 or 3 arguments but received: %d.\n" + SAMPLE_USAGE;
-    private static String SAMPLE_USAGE_WITH_INVALID_PROTOCOL = "Expected argument 2 to be one of 'mqtt', 'https', 'amqps' or 'amqps_ws' but received %s\n" + SAMPLE_USAGE;
-
-    private static IotHubClientProtocol protocol = IotHubClientProtocol.MQTT;
+    private static String SAMPLE_USAGE_WITH_INVALID_PROTOCOL = "Expected argument 2 to be one of 'mqtt', 'amqps' or 'amqps_ws' but received %s\n" + SAMPLE_USAGE;
```

Comment:
https is not supported

Reply:
what about mqtt_ws?

Reply:
That does not exist in the list so I did not test it. Do we support that? amqps_ws is supported though.

Reply:
Afaik methods should be supported over both tcp and ws, for mqtt (usually the feature is supported over both protocols, with proxy being the only feature/capability that is restricted to ws).

Which list doesn't mention mqtt_ws? Does it include a comment about the omission maybe?

Reply:
Line 26 is what I went by for testing.

Reply:
It is possible that the sample is just now showing usage of mqtt_ws. We can add it in a subsequent PR to all samples where it is relevant.